airflow/models/taskinstance.py

```diff
@@ -394,6 +394,7 @@ class TaskInstance(Base, LoggingMixin):
     queue = Column(String(256))
     priority_weight = Column(Integer)
     operator = Column(String(1000))
+    custom_operator_name = Column(String(1000))
```

Comment:
Let’s split changes related to ti to another PR as well, this involves migration and complicates the changes quite a lot.

Reply:
@uranusjr Sure, I'll move it to other PR

Reply:
Moved all the `taskinstance` related changes to PR - https://github.com/apache/airflow/pull/31852